README.md

```diff
@@ -1,9 +1,222 @@
 # ros_utilities
 Wrappers and other utilities for ROS2
 
+## Getting started
+
+### Process-wide APIs
+
+These APIs wrap [`rclpy`](https://github.com/ros2/rclpy) APIs to provide a simpler UX in common use cases.
+All APIs are thread-safe and exception-safe. Resource management is automatic. By default, callbacks are
+dispatched in the background, enabling both synchronous and asynchronous programming.
+
+These APIs are also quite handy to reconcile past [`rospy`](http://wiki.ros.org/rospy) experience with ROS 2.
+
+#### Setting up single node processes
+
+Just decorate your executable entrypoint (or `main` function) with `bdai_ros2_wrappers.process.main`:
+
+```python
+import logging
+import time
+
+import bdai_ros2_wrappers.process as ros_process
+
+import std_msgs.msg
+
+@ros_process.main()
+def entrypoint():
```

Comment:
The examples and explanations look great.

Going to ask for type hints just to keep it consistent with our code expectations.